### ISSUE TITLE
How can we get location without internet?

### ISSUE THREAD

**Issue by imvannet** [Lyokone/flutterlocation#833]

**Comment by Lyokone** [comment]
Yes. GPS gives you a position without an internet connection. Network based location (which uses wifi and cell towers for a rough fix) does need connectivity, but the GPS fix does not.

Closing since this is a usage question rather than a bug.
